jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -37,6 +37,8 @@ for indentation.
 $font-family: 'Ubuntu', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $font-family-mono: 'Ubuntu Mono', 'Ubuntu-Mono', 'Bitstream Vera Sans Mono', 'Courier New', Courier, monospace;
 
+$asset-path: 'juju-ui/assets/';
```

Comment:
There will not be a standard place for assets within the app? If that is
the case we will not be able to load any assets from CSS in the GUI. I will
work out a way of removing the font load on Vanilla if you want and then we
can inject the font into the the head of the index.

Or I'll play with base64'ing the font into Vanilla and see what size that
adds to it.

On Mon, 11 Apr 2016, 20:10 Jeff Pihach, <notifications@github.com> wrote:

> In jujugui/static/gui/src/app/assets/css/base.scss
> <https://github.com/juju/juju-gui/pull/1535#discussion_r59262942>:
>
> > @@ -37,6 +37,8 @@ for indentation.
> >  $font-family: 'Ubuntu', 'Helvetica Neue', Helvetica, Arial, sans-serif;
> >  $font-family-mono: 'Ubuntu Mono', 'Ubuntu-Mono', 'Bitstream Vera Sans Mono', 'Courier New', Courier, monospace;
> >
> > +$asset-path: 'juju-ui/assets/';
>
> This is interesting because the asset path in Juju 2.0 is actually
> different than the root path that the gui is loaded from so we actually
> have to get that value from juju on load. So this won't work for Juju 2.0
> story.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/juju-gui/pull/1535/files/3dde770631c35fce5169e284e81e1d7aa07ad97e#r59262942>
>